shared/src/profileData.ts

```diff
@@ -13,6 +13,9 @@ export const cannabisLicenseOptions = ["CONDITIONAL", "ANNUAL"] as const;
 export const carServiceOptions = ["STANDARD", "HIGH_CAPACITY", "BOTH"] as const;
 export const constructionOptions = ["RESIDENTIAL", "COMMERCIAL_OR_INDUSTRIAL", "BOTH"] as const;
 export const residentialConstructionOptions = ["NEW_HOME_CONSTRUCTION", "HOME_RENOVATIONS", "BOTH"] as const;
+export const employmentPersonnelServiceOptions = ["JOB SEEKERS", "EMPLOYERS"] as const;
```

Comment:
[pebble] can we make this `JOB_SEEKERS` instead to follow the pattern we use in the others for types with spaces in them.

Reply:
done